x-pack/plugins/lens/public/xy_visualization/xy_config_panel.tsx

```diff
@@ -324,9 +324,29 @@ export function XyToolbar(props: VisualizationToolbarProps<State>) {
               {isFittingEnabled ? (
                 <EuiFormRow
                   display="columnCompressed"
-                  label={i18n.translate('xpack.lens.xyChart.missingValuesLabel', {
-                    defaultMessage: 'Missing values',
-                  })}
+                  label={
+                    <EuiToolTip
+                      delay="long"
+                      position="top"
+                      content={i18n.translate('xpack.lens.xyChart.missingValuesLabelHelpText', {
+                        defaultMessage: `Lens provides a set of fitting functions to apply specific behaviour if 
+                        there are missing values in an area or line chart. Missing values are drawn using dotted 
+                        lines, and will not fill missing values at the left side of the chart.`,
+                      })}
+                    >
+                      <span>
+                        {i18n.translate('xpack.lens.xyChart.missingValuesLabel', {
+                          defaultMessage: 'Missing values',
+                        })}{' '}
+                        <EuiIcon
+                          type="questionInCircle"
+                          color="subdued"
+                          size="s"
+                          className="eui-alignTop"
+                        />
+                      </span>
+                    </EuiToolTip>
+                  }
```

Comment:
Small issue, but perhaps consider changing the current `label` prop to the following. It keeps the tooltip trigger on only the icon and removes the delay, which may be less annoying for users. This change will require that `EuiIconTip` also be imported.

```suggestion
                  label={
                    <>
                      {i18n.translate('xpack.lens.xyChart.missingValuesLabel', {
                        defaultMessage: 'Missing values',
                      })}{' '}
                      <EuiIconTip
                        color="subdued"
                        content={i18n.translate('xpack.lens.xyChart.missingValuesLabelHelpText', {
                          defaultMessage: `Lens provides a set of fitting functions to apply specific behaviour if 
    there are missing values in an area or line chart. Missing values are drawn using dotted 
    lines, and will not fill missing values at the left side of the chart.`,
                        })}
                        iconProps={{
                          className: 'eui-alignTop',
                        }}
                        position="top"
                        size="s"
                        type="questionInCircle"
                      />
                    </>
                  }
```